### ISSUE TITLE
Stage 2/4: Data Preprocessing

### ISSUE THREAD

**Issue by Carmoruda** [Carmoruda/JetBrains-Academy-Projects#15]
## To-do list
- [ ] Exclude the `Time` column from the dataset
- [ ] Divide the dataset into two parts
- [ ] Create test and training set for the target and features datasets
- [ ] Scale the test and training sets using StandrdScaler() from the sklearn library
- [ ] Print the first 3 elements of the scaled target test dataset

## Description
Data preprocessing is a very important step in machine learning. It gives developers control over the data fed to the algorithm. Data preprocessing can include several steps, the main ones being **data cleaning** and **transformation**. Let's cover them step by step.

Data cleaning:

Data can be noisy and contain missing or meaningless values. Data cleaning helps by setting an example to the algorithm of what our data should look like. If that's your case, you could refer to the **regression analysis** to determine which dataset feature is the most useful and what the machine learning algorithm is trying to achieve.

Data Transformation:

Imagine, you're making research on fish weights. You gather a lot of data on average fish weight from different seas and oceans, but due to different metric systems, some of the results are in kilograms (kg), some are in pounds (Ibs). This data should be processed before any further steps. Normalization scales the data along a certain range to form correct models. Normalization is not the only method used in data transformation, there’s also **data aggregation** and **feature selection**.
## Objectives
In this stage you should:

1.  Exclude the `Time` column from the dataset;
2.  Divide the dataset into two parts. The `target` dataset contains the last column and the `features` dataset has the remaining columns;
3.  Create test and training set for the `target` and `features` datasets. Use the `train_test_split()` [function](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) with `test_size=0.2` and `random_state=1`;
4.  Scale the test and training sets using `StandrdScaler()` from the `sklearn` library. If you want to learn more about `StandardScaler()`, you can refer to the [Sklearn Documentation](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html);
5.  Print the first 3 elements of the scaled `target` test dataset.
## Example
*Input: the dataset with information about anonymized credit card transactions, labeled as fraudulent or genuine.*

*Output: the first 3 elements of the scaled dataset in the following form:*
```
[[-0.72979737  0.54096286 -0.58281915  0.71493311  0.56751041 -0.18430649
   0.62295351  0.35232832 -0.97004687 -0.06336563  0.41262965  0.77389237
  -0.23123522  1.14025515 -1.03913106 -0.90290497  0.17787971  0.24642771
   1.31443245 -0.25582611  0.36909942  0.92298265 -0.65451586  1.2322706
  -0.19901331 -1.04698153 -1.06465901 -0.81976888 -0.12201909]
 [-0.56688802 -0.20199922  1.05053597 -1.63261088 -0.34735803 -0.04690893
   0.49426948  0.05068242 -0.92831065 -0.38312569 -0.88479198 -0.64073845
   0.65163304 -0.75697007 -0.39753556  1.67486088 -0.21195195 -1.63144656
   0.05057018  0.86220797  0.01970921 -0.47202744  0.31708313  1.0991614
   1.08071579 -0.96657544  0.48874755  0.41304922  0.35646337]
 [ 1.03799724  0.19750849 -1.00705899  0.42656459  0.19148847 -1.04757187
   0.32207181 -0.35358904  0.44723765 -0.58695506 -0.1386906   1.05306918
   1.32944309 -1.21024693  0.15784578 -0.02088215  0.67349466 -0.35178256
  -0.02082534 -0.16059329 -0.41427124 -0.88543782  0.41682997 -0.09145081
  -0.13293825 -1.22308831  0.03300227 -0.05841223 -0.33441019]]
```
